src/schema/v2/partner/partner.ts

```diff
@@ -233,6 +246,54 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      artistsWithAlertCounts: {
```

Comment:
If this is a connection we should make it known in the name! So, `artistsWithAlertCountsConnection`

Reply:
Excellent eyes 🦅 !